docs/index.rst

```diff
@@ -76,7 +78,7 @@
 
 .. raw:: html
```

Comment:
do we still need HTML in the page?

Reply:
Yes, we managed to remove one html tag but the issue is still open: https://github.com/scylladb/sphinx-scylladb-theme/issues/217